eZ/Publish/API/Repository/Tests/SearchServiceDeleteTranslationTest.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+/**
+ * This file is part of the eZ Publish Kernel package.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Publish\API\Repository\Tests;
+
+use eZ\Publish\API\Repository\Tests\SetupFactory\LegacyElasticsearch;
+use eZ\Publish\API\Repository\Values\Content\Content;
+use eZ\Publish\API\Repository\Values\Content\Query;
+use eZ\Publish\API\Repository\Values\Content\Query\Criterion;
+use eZ\Publish\API\Repository\Values\Content\Search\SearchResult;
+
+/**
+ * Test case for delete content translation with the SearchService
+ *
+ * @see \eZ\Publish\API\Repository\SearchService
+ * @group integration
+ * @group search
+ */
+class SearchServiceDeleteTranslationTest extends BaseTest
+{
+    /**
+     * @throws \ErrorException
+     */
+    public function setUp()
+    {
+        $setupFactory = $this->getSetupFactory();
+
+        if ($setupFactory instanceof LegacyElasticsearch) {
+            $this->markTestIncomplete('Not implemented for Elasticsearch Search Engine');
+        }
+
+        parent::setUp();
+    }
+
+    /**
+     * @param array $languages
+     * @return \eZ\Publish\API\Repository\Values\Content\Content
+     *
+     * @throws \eZ\Publish\API\Repository\Exceptions\BadStateException
+     * @throws \eZ\Publish\API\Repository\Exceptions\ContentFieldValidationException
+     * @throws \eZ\Publish\API\Repository\Exceptions\ContentValidationException
+     * @throws \eZ\Publish\API\Repository\Exceptions\InvalidArgumentException
+     * @throws \eZ\Publish\API\Repository\Exceptions\NotFoundException
+     * @throws \eZ\Publish\API\Repository\Exceptions\UnauthorizedException
```

Comment:
AFAIR Some of those throw common ForbiddenException. Replace them with that please.